### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by ravichdev**
We are very excited about this v0.2.0 release of the official Material Design plugin and theme for WordPress! Lots of work went into this release, and it is loaded with new features and improvements in usability.

For the full list of changes in this release, please see the [0.2.0 milestone](https://github.com/material-components/material-design-for-wordpress/issues?q=is%3Aopen+is%3Aissue+milestone%3A0.2.0).

The key highlighted features in this release include:

- **Settings Page** with options to enable auto updates for Google Fonts and Icons, Material Design plugin and theme (using the WordPress core auto-update API).
- **Curated Card Collection Block** supports custom post types and an improved UI in the block editor.
- **Gallery Block** with device specific (desktop, tablet & mobile) columns and padding options.
- **Button Block** with a tooltip text option for the icon button style.

## Changelog

- New feature: Tooltip option to Button block with icon style.
- New feature: Device specific column options for gallery block.
- New feature: Custom post type support for the "Curated card collection" block.
- New feature: Settings page with options to toggle auto updates and save API key.
- New feature: Option to enable auto updates for Google Icons.
- New feature: Option to enable auto updates for Google Fonts.
- New feature: Option to enable auto updates for Material Design plugin and theme using WordPress core auto updates API.
- Update: "Curated card collection" block display in the Block Editor to show a "Preview" button which links to the post.
- Update: Card header image to use an `img` tag to leverage WordPress core `srcset`.
- Fix: Bug in the Block Editor card block where the card header image cannot be removed.

## Props
Thanks to the many contributors who made this release possible through work on development, design, testing, project management, and more:

@ravichdev, @emeaguiar, @csossi, @jauyong, @mattchungxwp, @dawidmlynarz, @rodydavis 

---

- [ ] Write a complete description
- [x] Update changelog
- [x] Add contributors
- [ ] Merge & tag a release

### CODE REVIEW

**Comment by emeaguiar** [comment]
@googlebot I consent.

**Comment by kasparsd** [comment]
@googlebot I consent.

**Comment by rodydavis** [comment]
@googlebot I consent.

File: .gitignore [hunk 1]
```diff
@@ -2,15 +2,20 @@
...
+theme/assets/js/*
 bin/local-dev/mysql/
 bin/local-dev/wordpress/html/
 build/
 node_modules/
-tests/coverage
 vendor/
+plugin/tests/coverage
+plugin/composer*
+theme/composer*
```
Comment (rodydavis) [inline .gitignore:21]: Need a new line at the end of this file

File: phpcs-theme.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	</rule>
+	<rule ref="PHPCompatibilityWP" />
+
+	<config name="testVersion" value="5.6-" />
+
+	<file>.</file>
+
+	<exclude-pattern>*/assets/*</exclude-pattern>
+	<exclude-pattern>*/tests/coverage/*</exclude-pattern>
+</ruleset>
```
Comment (rodydavis) [inline phpcs-theme.xml:51]: New line needed at this end of this file

**Review by rodydavis — CHANGES_REQUESTED** [review]
Looks good but there are a few files that need a new line at the end. @ravichdev

File: plugin/assets/src/block-editor/blocks/image-list/block.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+		},
+		"linkTo": {
+		   "type": "string",
+		   "default": "media"
+		}
+	},
+	"supports": {
+		"align": true
+	}
+}
```
Comment (rodydavis) [inline plugin/assets/src/block-editor/blocks/image-list/block.json:101]: Needs a blank line at the end

**Review by rodydavis — CHANGES_REQUESTED** [review]
Missing a blank line as per the style guide

**Comment by ravichdev** [comment]
@rodydavis I've added description for the release, please review.

File: plugin/assets/src/block-editor/blocks/image-list/block.json (hunk 3 above)
Comment (rodydavis) [reply plugin/assets/src/block-editor/blocks/image-list/block.json]: @ravichdev  can you push this change?

**Review by rodydavis — APPROVED** [review]
LGTM

**Pull request merged**